File: devops-scm-api/src/main/java/com/tencent/devops/scm/api/pojo/Comment.java
package com.tencent.devops.scm.api.pojo;

import java.util.Date;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * 评论
 */
@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class Comment {
    private Long id;
    private String body;
    private String link;
    private User author;
    private Date created;
    private Date updated;
    private String type;
}

File: devops-scm-api/src/main/kotlin/com/tencent/devops/scm/api/pojo/Comment.kt
package com.tencent.devops.scm.api.pojo

import java.time.LocalDateTime

data class Comment(
    val id: Long,
    val body: String,
    val link: String,
    val author: User,
    val created: LocalDateTime,
    val updated: LocalDateTime,
    val type: String
)
